File: co_occur_vis.py
import numpy as np

co_occur = np.loadtxt('sorted_co_occur_count_freq.txt',dtype=np.int)
with open('co_occur_freq_vis.html','wb') as f:
	f.write('<html><head></head><table>\n')
	for i in range(5000):
		f.write('<tr><td><img src="base_address/'+str(co_occur[i][0])+'.jpg"></td><td><img src="base_address/'+str(co_occur[i][1])+'.jpg"></td><td>'+str(co_occur[i][2])+'</td></tr>\n')
	f.write('</table></html>')

File: co_occur_vis.m
fnew = fopen('co_occuring_words.txt','a+');
fhtml = fopen('co_occuring_vis.html','a+');
fprintf(fhtml,'<html><head></head><table>\n');
for i = 0:5000
	fprintf(fnew,'%d %d %d\n',co_occur_shorted(end - i,1),co_occur_shorted(end - i,2),co_occur_shorted(end - i,3));
	fprintf(fhtml,'<tr><td><img src="base_address/%d.jpg"></td><td><img src="base_address/%d.jpg"></td><td>%d</td></tr>\n',co_occur_shorted(end - i,1),co_occur_shorted(end - i,2),co_occur_shorted(end - i,3));
end

fclose(fnew);
fclose(fhtml);